modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java

```diff
@@ -187,10 +335,264 @@ private void takeOverActiveActor() {
      */
     private void stepDownActiveActor() {
         isActiveActor = false;
+
+        stopUpdater();
     }
 
     @TestOnly
     boolean isActiveActor() {
         return isActiveActor;
     }
+
+    /**
+     * Starts a dedicated thread to renew or assign leases.
+     */
+    private void startUpdater() {
+        //TODO: IGNITE-18879 Implement lease maintenance.
+        updater = new Thread(NamedThreadFactory.threadPrefix(clusterService.topologyService().localMember().name(), "lease-updater")) {
```

Comment:
shouldnt it be a part of maintenance?

Reply:
Now, we have no a dedicate part to crewate a lease and maintainence. Beside this it ia a part of general workflow.